### PR TITLE
[IN-0] Fix script to only have a test name once in the list

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I updated the corresponding tests if there were any.
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## UPCOMING
 
 ## `v2021_02_18`
+* `Only trigger said tests once`
```
Comment (peterheja-bitrise) [inline CHANGELOG.md:4]: Should be split into v2021_02_18_2

**Review by peterheja-bitrise — CHANGES_REQUESTED** [review]

**Review by peterheja-bitrise — DISMISSED** [review]
There is no _2 behind it, but I'm not going to RC this one once again.

**Review by peterheja-bitrise — APPROVED** [review]
LGTM

**Review by bitrise-tom — APPROVED** [review]
I did point this out on the last PR :)

**Pull request merged**